tests/List.test.jsx

```diff
@@ -3,24 +3,6 @@ import { MemoryRouter } from 'react-router-dom';
 import { List } from '../src/views/List';
 import { mockShoppingListData } from '../src/mocks/__fixtures__/shoppingListData';
 
-beforeEach(() => {
```

Comment:
Is there a reason why you removed this whole `beforeEach`?

You should only have to remove line 21, and the test should pass.